docs/source/user-guide/tutorials/build-r-pkgs.rst

```diff
@@ -0,0 +1,296 @@
+======================================
+Building R packages with skeleton CRAN
+======================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+This tutorial describes how to quickly build
+an R-language package on macOS for an R module
+that is already available on CRAN.
+
+You will build a simple package that can be
+installed in any conda environment of the same
+R version as your root environment. The tutorial
+will also tell you how to build the dependencies
+that may arise while building the package.
+
+Who is this for?
+================
+This tutorial is for macOS users who wish to
+build an R-language package from CRAN.
+No prior knowledge of conda build or conda recipes
+is required.
+
+
+.. _before-you-start3:
+
+Before you start
+================
+
+Before you start, check the :doc:`prerequisites <index>`.
+
+.. _conda-build-cran:
+
+Building a simple package with conda skeleton CRAN
+==================================================
+
+The conda skeleton command picks up the CRAN package
+metadata and prepares the conda build recipe. The final
+step is to build the package itself and install it into
+your conda environment.
+
+It is easy to build a skeleton recipe for any R package that is hosted on CRAN.
+In this section you are going to use conda skeleton to generate a conda recipe,
+which informs conda build about where the source files are located and how to
+build and install the package.
+
+You'll be working with a package that is hosted on CRAN named fansi_,
+a tool that accounts for the effects of ANSI text formatting control sequences.
+
+.. _fansi: https://github.com/conda-forge/r-fansi-feedstock
+
+First, in your user home directory, run the conda skeleton command:
+
+.. code-block:: bash
+
+    conda skeleton cran fansi
+
+The two arguments to ``conda skeleton`` are the hosting location,
```

Comment:
technically, it's not really the hosting location, but rather the *kind* of hosting location.  CRAN is not so much a place, it is a place that behaves a particular way.  Other places can behave that same way, and those are valid hosting locations, too.

Reply:
I changed it to "type of hosting location"